### DISCUSSION TITLE
Link and content disclaimer

### DISCUSSION THREAD

**Discussion by MikeMcC399** [corona-warn-app/cwa-website#3477] [General] [answered]
If it is planned to keep the website https://www.coronawarn.app/ online with unmaintained content and unmaintained links then it would be a good idea to include disclaimers on the site (possibly also in the FAQs) explaining that the content and links can no longer be relied on, as appropriate, depending on what level of maintenance is going to be kept up after end-of-development.

**Comment by Ein-Tim** [comment, marked as answer]
I have forwarded this suggestion!

**Comment by MikeMcC399** [comment]
This can be considered done as there is a header on the site and many of the FAQ entries have notices on them.
